code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -68,6 +68,18 @@
 	skirts["jumper dress"] = /obj/item/clothing/under/skirt/jumper_highcut
 	gear_tweaks += new /datum/gear_tweak/path(skirts)
 
+/datum/gear/uniform/suitskirt
+	display_name = "skirtsuit (recolourable)"
+	description = "A recolourable skirtsuit."
+	path = /obj/item/clothing/under/suit_jacket/skirtsuit/color
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
+
+/datum/gear/uniform/command_suitskirt
+	display_name = "command skirtsuit"
+	description = "A Skirtsuit for staff in a command role."
```

Comment:
```suggestion
	description = "A skirtsuit for staff in a command role."
```

Reply:
The rules of the english language elude me I am terribly sorry.
![grafik](https://user-images.githubusercontent.com/30761807/153257538-9d94849e-1bd1-4582-b509-700127a3c36e.png)